src/test/java/org/example/pagefunctions/AdminPage.java

```diff
@@ -151,7 +151,7 @@ public void searchSystemUser(WebDriver driver)
         findButtonClick(objRepo.searchButton);
         sleep(1);
         highlightElements(driver,objRepo.systemUserLink);
-        Assert.assertTrue(objRepo.systemUserLink.isDisplayed());
+        //Assert.assertTrue(objRepo.systemUserLink.isDisplayed());
```

Comment:
I wanted to comment on the specific lines that you need to update, However, you already merged that part of the code, so Github doesn't give me option to do it. I only can comment on the lines that are pushed with this pull request. So I am leaving the comment here. In this class, your code is not reader friendly, you did not use any spaces at all between lines. You are not asked to use spaces everywhere, which is not a good approach, but at lease use some space to separate different parts of the code. Another thing is that I recommend you to use "//" to leave comments just above the actions you're performing. So people can understand easily what you are trying to do there.

Reply:
Third thing is that your methods are too big. For example, validateAdminPageText method is huge. Lots of repeated lines are there:

Reporter.log("Validating all the links on the Admin page");
        waitForElementVisible(driver,objRepo.adminLink);
        highlightElements(driver,objRepo.adminLink);
        Assert.assertTrue(objRepo.adminLink.isDisplayed());
        findButtonClick(objRepo.adminLink);
        Reporter.log("Validating user management list on the Admin page");
        waitForElementVisible(driver,objRepo.userManagementLink);
        highlightElements(driver,objRepo.userManagementLink);
        Assert.assertTrue(objRepo.userManagementLink.isDisplayed());
        findButtonClick(objRepo.userManagementLink);

these 5 lines/steps are repeated so many times, so why don't you just creat a method which contains these 5 lines of code, and call it in this validateAdminPageText method? You can update all other methods in this manner. Check your code and find out repeated lines/code and update them.